tests/python/unittest/test_gluon_data_vision.py

```diff
@@ -229,6 +229,22 @@ def test_transformer():
     transform(mx.nd.ones((245, 480, 3), dtype='uint8')).wait_to_read()
 
 
+@with_seed()
+def test_randomtransforms():
```

Comment:
add an underscore between random and transforms

Reply:
Done